File: BaekJoon/1152_단어의 개수.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s = sc.nextLine();
		int sum = 0;
		int check = 0;
		for (int i = 0; i < s.length(); i++) {
			if (s.charAt(i) == ' ') {				 				
				sum++;
			}			
		
		}		
		if (s.charAt(0) == ' ') sum--; 
		if (s.charAt(s.length()-1) == ' ') sum--;
		System.out.println(sum+1);
	}
}

File: BaekJoon/1152_단어의 개수.kt
import java.util.*
fun main(args: Array<String>) {
    var sc:Scanner = Scanner(System.`in`)
    var str: String = sc.nextLine()
    var check = 0
    for (i in 0 until str.length) {
        if (str.get(i) == ' ') {
            check++
        }
    }
    if (str.get(0) == ' ') check--
    if (str.get(str.length-1) == ' ') check--
    print(check+1)
}
